app/components/reject-message-field.js

```diff
@@ -1,31 +1,43 @@
-import Ember from 'ember';
+import Ember from "ember";
 import { translationMacro as t } from "ember-i18n";
 
 export default Ember.Component.extend({
   disabled: false,
   placeholderText: t("reject.message_placeholder"),
   i18n: Ember.inject.service(),
   store: Ember.inject.service(),
+  attributeBindings: ["value"],
 
-  rejectMessage: Ember.computed('selectedId', {
+  valueChanged: Ember.observer("rejectMessage", function() {
+    this.get("setMessage")(this.get("rejectMessage"));
+  }),
+
+  rejectMessage: Ember.computed("selectedId", {
     get: function() {
-      var reasonRecord = this.get('store').peekRecord('rejection_reason', this.get('selectedId'));
-      var reason = reasonRecord && reasonRecord.get('name');
+      var reasonRecord = this.get("store").peekRecord(
+        "rejection_reason",
+        this.get("selectedId")
+      );
+      var reason = reasonRecord && reasonRecord.get("name");
```

Comment:
If possible we can start refactoring old codes with `let` and `const` wherever possible.

Reply:
Sure 👍